### PR TITLE
Refactor: rename carplay message to command, encapsulat touch logic in hook

### PR DESCRIPTION
**Pull request by gozmanyoni**

### CODE REVIEW

File: src/web/tsconfig.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "lib": ["es2023", "DOM", "WebWorker"],
```
Comment (gozmanyoni) [inline src/web/tsconfig.json:4]: formatter change

**Review by steelbrain — APPROVED** [review]
Makes sense, go for it

File: examples/carplay-web-app/src/App.tsx [hunk 2]
```diff
@@ -1,15 +1,11 @@
 import { useCallback, useEffect, useMemo, useState } from 'react'
 import { RotatingLines } from 'react-loader-spinner'
 import './App.css'
-import {
-  TouchAction,
-  findDevice,
-  requestDevice,
-  DongleConfig,
-} from 'node-carplay/dist/web'
+import { findDevice, requestDevice, DongleConfig } from 'node-carplay/dist/web'
```
Comment (steelbrain) [inline examples/carplay-web-app/src/App.tsx:4]: Was tired of seeing `dist/web` so opened https://github.com/rhysmorgan134/node-CarPlay/pull/44 to make it just `/web` so it looks more refined as a package

**Comment by rhysmorgan134** [comment]
Looks good

**Pull request merged**